loky/backend/semaphore_tracker.py

```diff
@@ -203,7 +203,6 @@ def main(fd):
                 try:
                     sem_unlink(name)
                     if VERBOSE:  # pragma: no cover
-                        name = name.decode('ascii')
                         sys.stderr.write("[SemaphoreTracker] unlink {}\n"
                                          .format(name))
```

Comment:
maybe change that to `name.decode('ascii')`

Reply:
Indeed.

Reply:
Actually no: `name` is now expected to be a `str` in Python 3 and a `unicode` in Python 2.